scripts/zkai_d64_recursive_pcd_aggregation_feasibility_gate.py

```
@@ -0,0 +1,673 @@
+        attempt.get("go_criterion"),
+        recursive_or_pcd_attempt()["go_criterion"],
+        "recursive or PCD go criterion",
+    )
+    expect_equal(
+        attempt.get("blocked_before_metrics"),
+        recursive_or_pcd_attempt()["blocked_before_metrics"],
+        "blocked-before metrics",
+    )
+
+
+def _validate_case_metadata(payload: dict[str, Any], *, required: bool) -> tuple[int, int] | None:
```

Ruff (0.15.12)

[warning] Too many branches (15 > 12)

(PLR0912)